src/sentry/middleware/integrations/parsers/slack.py

```diff
@@ -82,19 +92,101 @@ class SlackRequestParser(BaseRequestParser):
     See: `src/sentry/integrations/slack/views`
     """
 
+    def build_loading_modal(self, external_id: str, title: str):
+        return {
+            "type": "modal",
+            "external_id": external_id,
+            "title": {"type": "plain_text", "text": title},
+            "blocks": [
+                {
+                    "type": "section",
+                    "block_id": "loading_block",
+                    "text": {"type": "mrkdwn", "text": "Loading..."},
```

Comment:
nit: i wonder if it's possible to have a loading indicator, that would be ideal lMO

Reply:
i looked for some built in block, there doesn't seem to be one. what we can do is have a static image that we render 👀

Reply:
nw this is just a nice to have